defaults/host-ents.go

```diff
@@ -0,0 +1,197 @@
+
+	hostProcessesNoneEntitlement  = entitlement.NewVoidEntitlement(HostProcessesNoneEntFullID, hostProcessesNoneEntitlementEnforce)
+	hostProcessesAdminEntitlement = entitlement.NewVoidEntitlement(HostProcessesAdminEntFullID, hostProcessesAdminEntitlementEnforce)
+)
+
+var (
+	allowedMounts = []specs.Mount{
+		{
+			Destination: "/proc",
+			Type:        "proc",
+			Source:      "proc",
+			Options:     []string{"nosuid", "noexec", "nodev"},
+		},
+		{
+			Destination: "/dev",
+			Type:        "tmpfs",
+			Source:      "tmpfs",
+			Options:     []string{"nosuid", "strictatime", "mode=755"},
+		},
+		{
+			Destination: "/dev/pts",
+			Type:        "devpts",
+			Source:      "devpts",
+			Options:     []string{"nosuid", "noexec", "newinstance", "ptmxmode=0666", "mode=0620", "gid=5"},
+		},
+		{
+			Destination: "/sys",
+			Type:        "sysfs",
+			Source:      "sysfs",
+			Options:     []string{"nosuid", "noexec", "nodev", "ro"},
+		},
+		{
+			Destination: "/sys/fs/cgroup",
+			Type:        "cgroup",
+			Source:      "cgroup",
+			Options:     []string{"ro", "nosuid", "noexec", "nodev"},
+		},
+		{
+			Destination: "/dev/mqueue",
+			Type:        "mqueue",
+			Source:      "mqueue",
+			Options:     []string{"nosuid", "noexec", "nodev"},
+		},
+	}
+)
+
+func isAllowedMount(mount specs.Mount) bool {
+	for _, allowedMount := range allowedMounts {
+		if reflect.DeepEqual(mount, allowedMount) {
+			return true
+		}
+	}
+
+	return false
+}
+
+func hostDevicesNoneEntitlementEnforce(profile secprofile.Profile) (secprofile.Profile, error) {
+	ociProfile, err := ociProfileConversionCheck(profile, HostDevicesNoneEntFullID)
+	if err != nil {
+		return nil, err
+	}
+
+	capsToRemove := []types.Capability{
+		CapSysAdmin,
+	}
+	ociProfile.RemoveCaps(capsToRemove...)
+
+	ociProfile.AppArmorSetup.Files.ReadOnly = append(ociProfile.AppArmorSetup.Files.ReadOnly, "/sys/**")
+	ociProfile.AppArmorSetup.Files.Denied = append(ociProfile.AppArmorSetup.Files.Denied, "/proc/kcore/**")
+
+	ociProfile.OCI.Linux.ReadonlyPaths = append(ociProfile.OCI.Linux.ReadonlyPaths, "/sys")
+	ociProfile.OCI.Linux.MaskedPaths = append(ociProfile.OCI.Linux.MaskedPaths, "/proc/kcore")
+
+	ociProfile.OCI.Mounts = allowedMounts
+
+	return ociProfile, nil
+}
+
+func hostDevicesViewEntitlementEnforce(profile secprofile.Profile) (secprofile.Profile, error) {
+	ociProfile, err := ociProfileConversionCheck(profile, HostDevicesViewEntFullID)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, mount := range ociProfile.OCI.Mounts {
+		mountPath := mount.Destination
+
+		if !isAllowedMount(mount) {
+			ociProfile.OCI.Linux.ReadonlyPaths = append(ociProfile.OCI.Linux.ReadonlyPaths, mountPath)
+		}
+	}
+
+	return ociProfile, nil
+}
+
+func removeReadOnlyFlagMount(mount specs.Mount) specs.Mount {
+	readWriteMount := mount
+	for index, option := range readWriteMount.Options {
+		if option == "ro" {
+			readWriteMount.Options = append(readWriteMount.Options[:index], readWriteMount.Options[index+1:]...)
```

Comment:
As discussed, mounts are RW by default and have the "ro" option if meant to be mounted as read-only so admins can now have sysfs etc.. mounted as RW. Will add docstring comment for this.